modules/HomePage/components/Posts/Posts.tsx

```diff
@@ -31,6 +40,35 @@ const Posts: FC<IPostFeed.IPosts> = ({ data }): ReactElement => {
     return null;
   };
 
+  const getMorePosts = async (): Promise<void> => {
+    const loggedInUser = getUserToken();
+    if (!loggedInUser) {
```

Comment:
If the user isn't logged in he will be redirected to login page before even reaching the client why the need to check again?

Reply:
I'm checking if the user is logged in but the token is expired, then no need to make an API call

Reply:
So of the token is expired the cookie will be automatically deleted?